helper/awsutil/generate_credentials.go

```diff
@@ -3,10 +3,15 @@ package awsutil
 import (
 	"fmt"
 	"net/http"
+	"os"
```

Comment:
Meta-comment: it would be nice to not merge this yet given that awsutil has been moved in the go-kms-wrapping branch into sdk/. The lib it depends on is internal only for the moment, but really it's just waiting for the PR in Vault to be okayed before it's moved to public. There are some other changes in there so rather than have to do a re-porting of these changes please just don't merge it yet until that patch lands.

Reply:
@jefferai this has already been merged. Can we move forward with this patch?

Reply:
We just checked in, and yes, that blocker has been resolved. I'm going to remove the "Do Not Merge" prefix from this, and assign it to myself for review.